lib/try-patch.rb

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
Given there's only one usage of this, I'd prefer to hold off on this. If you do have a larger block of code that uses it, then I'd be happy to include it.